src/algebra/ne_zero.lean

```diff
@@ -70,8 +70,11 @@ nat_of_injective $ no_zero_smul_divisors.algebra_map_injective R M
 lemma of_ne_zero_coe [has_zero R] [has_one R] [has_add R] [h : ne_zero (n : R)] : ne_zero n :=
 ⟨by {casesI h, rintro rfl, contradiction}⟩
 
-lemma not_dvd_char (R) [add_monoid R] [has_one R]
-  (p : ℕ) [char_p R p] (k : ℕ) [h : ne_zero (k : R)] : ¬ p ∣ k :=
+lemma not_dvd_char [add_monoid R] [has_one R] (p : ℕ) [char_p R p] (k : ℕ) [h : ne_zero (k : R)] :
```

Comment:
This should really be called `not_char_dvd`, right? Slipped through the review of the previous PR.